client/src/featureform/register.py

```diff
@@ -153,6 +189,31 @@ def df_transformation(self,
                           name: str = "",
                           description: str = "",
                           inputs: list = []):
+        """
+        Register a Dataframe transformation source. The local.df_transformation decorator takes the contents
+        of the following function and executes the code it contains at serving time.
+
+        The name of the function is the name of the resulting source.
```

Comment:
somewhat ambiguous, may imply reverse causality. Switch to mayb "the name of the function is passed to the resulting source as its own name" or something

Reply:
Clearer?